src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,176 @@
+package duke;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/** Duke class to encapsulate the behaviour of a task manager */
```

Comment:
Should this be a multi-line comment instead? See similar comment in `Deadline.java`.